packages/tools/examples/videoContourSegmentation/index.ts

```diff
@@ -52,6 +52,44 @@ const segmentIndexes = [1, 2, 3, 4, 5];
 const segmentVisibilityMap = new Map();
 let activeSegmentIndex = 1;
 
+const configuredTools = new Map<string, any>();
```

Comment:
@sedghi - I think this should be our standard way of defining sets of tools to include in a list - putting them into a map
allows iterating over them in the original organization, and getting the appropriate configuration out.